lib/msf/util/java_deserialization.rb

```diff
@@ -52,15 +55,16 @@ def self.ysoserial_payload(payload_name, command=nil)
         bytes[(length_offset-1)..length_offset] = length
       end
 
-      # Replace "ysoserial\/Pwner" timestamp string with randomness for evasion
+      # Replace "ysoserial\/Pwner" timestamp and "ysoserial" string with randomness for evasion
       bytes.gsub!(/ysoserial\/Pwner00000000000000/, Rex::Text.rand_text_alphanumeric(29))
```

Comment:
Unrelated to this PR, but there should be no reason to use a regex here.

```suggestion
       bytes.gsub!('ysoserial/Pwner00000000000000', Rex::Text.rand_text_alphanumeric(29))
```